source/module_esolver/esolver_dp.cpp

```diff
@@ -100,4 +122,85 @@ namespace ModuleESolver
         GlobalV::ofs_running << " !FINAL_ETOT_IS " << dp_potential * ModuleBase::Ry_to_eV << " eV" << std::endl;
         GlobalV::ofs_running << " --------------------------------------------\n\n" << std::endl;
     }
+
+    bool ESolver_DP::type_map(const UnitCell& ucell)
```

Comment:
I find a new function here, can you add UT for this function in this PR?

Reply:
All functions in module_esolver have no UTs, I think the UT may be added later, together with other functions in module_esolver.

Reply:
@hongriTianqi can you give your advice for topic of UTs?

Reply:
@dyzheng OK. @YuLiu98 I think it is necessary to add unit tests for the functions of ESolver_DP::Init and type_map, as they are newly added functions. By the way, it seems not difficult to add unit tests for them. Thanks!